.NET/Microsoft.Recognizers.Text.Sequence/Config/ISequenceConfiguration.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.Recognizers.Text.Sequence
+{
+    public interface ISequenceConfiguration
```

Comment:
Why do we need this interface and not just use Options?
If we need this, we can rename it to IOptionsConfiguration for consistency.